pkg/sdn/plugin/node.go

```diff
@@ -204,9 +202,23 @@ func (node *OsdnNode) Start() error {
 		}
 	}
 
-	// Wait for kubelet to init the plugin so we get a knetwork.Host
-	log.V(5).Infof("Waiting for kubelet network plugin initialization")
-	<-node.kubeletInitReady
+	log.V(5).Infof("Waiting for kubelet and network plugin initialization")
+	err = kwait.ExponentialBackoff(
```

Comment:
> the only way the condition wouldn't be true almost immediately is if the host is egregiously overloaded

that's true with the current implementation, but this code shouldn't assume that order, or that no other long-running op will insert itself prior to runtime initialization